### ISSUE TITLE
depend on module not on CPAN

### ISSUE THREAD

**Issue by chorny** [ebassi/perl-Glib-JSON#1]
CPAN has no Glib::IO

**Comment by ebassi** [comment]
Yep, it would be great to be able to do that. Sadly, I don't maintain `Glib::IO`, and it hasn't seen much activity. I was planning to have a look at it, and maybe take over its development.

**Comment by chorny** [comment]
CPAn distibution should mention how to install Glib::IO
